SPECS/libical/libical.spec

```diff
@@ -4,7 +4,7 @@ Distribution:   Mariner
 Summary:	Reference implementation of the iCalendar data type and serialization format
 Name:		libical
 Version:	3.0.9
-Release:	4%{?dist}
+Release:	5%{?dist}
```

Comment:
Release bumps should be accompanied with an entry in the `%changelog` section

Reply:
changelog updated